db/seeds.rb

```diff
@@ -213,6 +217,67 @@
 )
 ##############################################################################
 
+Rails.logger.info "9/10 Devvy Ruxpin"
```

Comment:
What if, instead of having "Devvy Ruxpin" as a seed and resort to the hack of having the sign in as a user, all the following code would moved to a regular rake task that can only be run in development mode and that, by passing the username or user id, would do all the "followings" and create all the articles?

This way you don't need any hack. If you want a regular seeded DB you just run the normal seed, if you want more data you increase it with the multiplication factor and if you need to follow everything you just run a rake task meant for dev purposes.

Reply:
An alternative to the `quickin` would be to use a gem such as [Pretender](https://github.com/ankane/pretender) which would only work within the admin panel.

I actually don't know if an admin panel exists or how to access it in this codebase which is why I had not suggested this previously. 

I don't understand the objections to increasing the amount of data as currently the seed data is insufficent for general Q&A.

Reply:
It's not an objection, I'm saying that if you make it parameterized the user can decide how much seed data they want and it's future proof :) You can still raise the default multiplier to have a better seed data default.

I'm not familiar with Pretender.

There's an admin, it's under `/admin`. There are also internal pages which I think it's a "side admin", see the routes under `/internal`

Reply:
Why not just give them the right amount of seed data to test the app without configuration? In order to Q&A various pagination you need more seed data.

Reply:
Sounds like neither of you object to making the amount of data configurable, just what the default quantity of data should be. @omenking wants to increase the default quantity. @rhymes, are you ok with increasing that amount, or do you want the current quantity to be the default? (I agree with @omenking that the amount of seed data should be sufficient to exercise paging.)

Reply:
@keithrbennett I still think it should be a configurable param, we can also tweak the default later